### ISSUE TITLE
[Feature Request] Tmoproray Compatibility mode for general modules before #9 land

### ISSUE THREAD

**Issue by nongoal** [LSPosed/LSPosed#1926]
### Is your feature request related to a problem?/你的请求是否与某个问题相关？

The problem has been reported several times. I agree with you to push the adoption of module scope, but due to the slow progress of #9 , if there are no API to adopt, there are no means to push it. Except broken all the general modules that not designed for specific applications currently.

### Describe the solution you'd like/描述你想要的解决方案

Temporarily add a compatibility mode for general modules to work correctly (this is not 'select all' but global injection, ex. the core feature of https://github.com/M66B/XPrivacyLua is auto restrict new installed applications).
You can remove it once #9 land to push developers to adopt the new API, that will make sense then.

**Comment by vvb2060** [comment]
xprivacylua should hook system, not each application.

**Comment by yujincheng08** [comment]
#9 is not for "select all"

**Comment by nongoal** [comment]
> xprivacylua should hook system, not each application.

@vvb2060 It could be implemented by hook system, but hook each application is also OK (though may be not the best way).This is just an example, I mean all the modules that not designed for specific applications, include everyone else waiting for #9 



> #9 is not for "select all"

@yujincheng08 "select all" is just a description of compatibility mode before #9 land. Only with #9 general modules can adopt to work properly without "select al" and global injection.

**Comment by vvb2060** [comment]
Apps can use native code or sub-processes to get any info they want without java. Hook app processes to get privacy and security is cheating yourself.

**Comment by nongoal** [comment]
> Apps can use native code or sub-processes to get any info they want. Hook app processes to get privacy and security is cheating yourself.

XprivacyLua is good at blocking well known tracking framework, it's the same thing as using an adblocker to block trackers, since website can report information to any domain. This is off topic, I just want to say #9 takes too long to wait so if it's so hard to implement we need a temporary solution.
